package.json

```diff
@@ -45,7 +45,7 @@
     "node": ">=14"
   },
   "dependencies": {
-	  "notion": ""
+    "node-fetch": "^2.6.7"
```

Comment:
this can't be a hard dependency as the calling code may already have fetch or be using another version - instead use fetch via @seneca/provider utility functions -see https://github.com/senecajs/seneca-tangocard-provider/blob/main/src/tangocard-provider.ts for example

Reply:
Agreed. But `makeUtils` from `@seneca/provider` doesn't support `patchJSON` - that is another PR we may need to open.